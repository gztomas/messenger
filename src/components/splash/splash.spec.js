import React from 'react';
import { shallow } from 'enzyme';

import Splash from './splash';

describe('Splash', () => {
  const props = {};

  it('should render properly', () => {
    const wrapper = shallow(<Splash {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
