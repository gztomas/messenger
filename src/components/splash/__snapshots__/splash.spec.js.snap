// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash should render properly 1`] = `
<splash__Container>
  <Card
    cover={<ForwardRef(splash__StyledLogo) />}
    hoverable={true}
    style={
      Object {
        "width": 240,
      }
    }
  >
    <Meta
      description="This is just the boilerplate"
      title="Messenger App"
    />
  </Card>
</splash__Container>
`;
